src/npr/snakehelper.py

```diff
@@ -66,8 +67,152 @@ def flatten_irreg_lists(nested_list):
     else:
         return [nested_list]
 
+def run_command(cmd,logf):
+    """
+    run command and print stderr to log file logf
+    """
+    print('[yellow] {} [/yellow]'.format(cmd))
+    with open(logf, 'a') as f:
+        f.write("### command: ###\n")
+        f.write(cmd + '\n')
+
+    # run 
+    try:
+        with open(logf, "a") as f:
+            res = sp.run(cmd, stderr=f, check=True, shell=True)
+    except sp.CalledProcessError as e:
+        print("[red] Failed with return code [/red]", e.returncode)
+        print(res)
+        print("[red] Check also file {} [/red]".format(logf))
+        sys.exit(1)
+
+def guppy2dorado(model_name):
+    """
+    This is an effort to translate model names
+    Currently the model is encode in the report*.json or can be obtained from
+    >pod5 inspect debug <pod5>
+    However, those models follow guppy naming conventions and have to be translated to dorado
+    Below is a brute force translation which should work for most runs until it fails
+    It will ikely have to be extended
+    Notice that the there is also a limited set of dorado models that will need update
+    """
+    new_name = model_name
+    if re.match(r'^rna', model_name):
```

Comment:
Not clear for me either, but for dna_r10.4.1* only 400bps is relevant since 260bps is deprecated. For dna_r9.4.1* speed does not appear in the model name. For rna* there are really only two choices. Once I know what will discriminate rna002_70bps* vs rna004_130bps, I  will implement here. For now I rely on dorado exiting if the choice should be wrong.